std_msgs/String move_group
geometry_msgs/PoseStamped target_pose
std_msgs/Float32 velocity_scaling_factor
std_msgs/Int32 max_attempts
std_msgs/Float32 max_planning_time
---
std_msgs/Bool success
